resources/manifests/pod-checkpointer-cluster-role.yaml

```diff
@@ -9,3 +9,14 @@ rules:
       - nodes/proxy
     verbs:
       - get
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
```

Comment:
this should be `ClusterRole`

Reply:
Oh, cool. Can you explain why? I don't follow (sorry, very new in RBAC :))

Reply:
It seems to work as just `Role`. I guess if you make it `ClusterRole`, you can create a binding to it in other namespaces as well then?

Reply:
Yes, I don't see a reason to use `ClusterRole` (we don't want to reduce boilerplate by using this in other namespaces, even allow anyone to use this in other namespace by creating another binding...)